src/main/java/engine/property/manager/PropertyManager.java

```diff
@@ -43,7 +43,11 @@ public static void setProperty(String systemProperty, String systemStringValue)
     }
 
     public static BrowserName getBrowserName() {
-        return BrowserName.valueOf(getProperty(PropertyKeys.BROWSER_NAME));
+        String browserNameProperty = getProperty(PropertyKeys.BROWSER_NAME);
```

Comment:
@jakub-krzysiek @fnfldx @jakub-krzysiek @ilku12 do you approve this change?

Reply:
@Fishgame1 Nope - as discussed I think it'd be better for test to throw an error if browser name in properties file is empty instead of running test on certain browser in this case

Reply:
@TOM-AN1 I think you can revert those chagnes

Reply:
The changes was reverted.

Reply:
@TOM-AN1 Can you check again if these changes were reverted correctly?